pkg/scheduling/volumeusage.go

```diff
@@ -130,22 +123,24 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 			return driverName, nil
 		}
 	}
-	if storageClassName != "" {
-		driverName, err := driverFromSC(ctx, kubeClient, storageClassName)
-		if err != nil {
-			return "", err
-		}
-		if driverName != "" {
-			return driverName, nil
-		}
+
+	driverName, err := driverFromSC(ctx, kubeClient, storageClassName)
+	if storageClassName == "" || errors.IsNotFound(err) {
```

Comment:
This feels like a bunch of extra checking for empty strings and sees odd that we are doing a check both for the empty string AND also that the error is a NotFound error. Seems like there's a way that we could clean this error handling logic up a bit

Reply:
Agreed, this could be cleaned up by having `driverFromSC` return a not found error if the storage class is empty.

Reply:
On second thought, its probably best to handle this as a separate error.